### ISSUE TITLE
TypeError: Cannot set property 'enabled' of undefined

### ISSUE THREAD

**Issue by feross** [webtorrent/webtorrent-desktop#1315]
**What version of WebTorrent Desktop?** (See the 'About WebTorrent' menu)

0.19.0

**What operating system and version?**

win32

**What did you do?**

```
Processes: main, platforms: win32, versions: 0.18.0 0.19.0
TypeError: Cannot set property 'enabled' of undefined
    at Object.setWindowFocus (...\build\main\menu.js:50:39)
    at BrowserWindow.onWindowFocus (...\build\main\windows\main.js:258:8)
    at emitOne (events.js:101:20)
    at BrowserWindow.emit (events.js:188:7)
```

This is the second most common error in the telemetry since the release of 0.19. cc @codealchemist - thoughts?

**Comment by codealchemist** [comment]
I wasn't able to reproduce, but found that I was missing an initial state for `highestPlaybackPriority`, which is used to set a checkbox on Preferences page. It this doesn't fix the issue we should continue looking for a way to reproduce it and I'll work on a fix.

Thanks!

**Comment by codealchemist** [comment]
Interesting. Let's keep an eye on this one.
If it continues to happen I'd like to spend more time on it.

Thanks!

**Comment by feross** [comment]
This is our fifth-most common WebTorrent Desktop error. Adding it to the next sprint.

**Comment by feross** [comment]
Should be fixed by #1938. We'll find out after the next WebTorrent Desktop release. If not, we can re-open this issue.
